lib/compat/wordpress-5.9/block-template-utils.php

```diff
@@ -474,12 +473,8 @@ function _remove_theme_attribute_in_block_template_content( $template_content )
 			}
 		}
 
-		if ( $has_updated_content ) {
-			foreach ( $template_blocks as $block ) {
-				$new_content .= serialize_block( $block );
-			}
-
-			return $new_content;
+		if ( ! $has_updated_content ) {
+			return $template_content;
```

Comment:
There were some minor changes to this function in WordPress core, so I've replicated them for Gutenberg.

Reply:
In this branch there is no `foreach ( $template_blocks as $block )` which is something I see in the Core version. Is that right?

Reply:
Yeah, oops. I obviously wasn't thorough enough. Fixed in https://github.com/WordPress/gutenberg/pull/37137.